core/primitives-core/src/account.rs

```diff
@@ -20,6 +23,189 @@ pub struct Account {
     pub storage_usage: StorageUsage,
 }
 
+#[cfg(feature = "protocol_feature_add_account_versions")]
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, PartialEq, Eq, Debug, Clone)]
+pub struct AccountV1 {
+    /// The total not locked tokens.
+    #[serde(with = "u128_dec_format_compatible")]
+    pub amount: Balance,
+    /// The amount locked due to staking.
+    #[serde(with = "u128_dec_format_compatible")]
+    pub locked: Balance,
+    /// Hash of the code stored in the storage for this account.
+    pub code_hash: CryptoHash,
+    /// Storage used by the given account, includes account id, this struct, access keys and other data.
+    pub storage_usage: StorageUsage,
+}
```

Comment:
Maybe it makes sense to make fields private for this `AccountV1` and old account `Account`. It shouldn't affect anything in the code and will catch any potential bugs with the direct access to the fields for files where the feature might not be enabled.

Reply:
Done